specifications/api/swagger.yaml

```diff
@@ -681,15 +682,13 @@ responsesPostWallet: &responsesPostWallet
     schema: *ApiWallet
 
 responsesGetWallet: &responsesGetWallet
-  <<: *responsesErr400
   <<: *responsesErr404
   <<: *responsesErr406
   200:
     description: Ok
     schema: *ApiWallet
 
 responsesDeleteWallet: &responsesDeleteWallet
-  <<: *responsesErr400
   <<: *responsesErr404
   <<: *responsesErr406
   <<: *responsesErr423
```

Comment:
also don't know how to produce 423...

Reply:
I thought we would be more granular in the wallet error reporting but we are not there yet. I think for now, it'll just hang and wait for the resource to become available again.

Reply:
shall we remove it then?

Reply:
Oui.